src/main/java/nl/knaw/dans/migration/core/tables/ExpectedDataset.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2021 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.migration.core.tables;
+
+import nl.knaw.dans.migration.core.EasyFile;
+import nl.knaw.dans.migration.core.FileRights;
+import nl.knaw.dans.migration.core.ManifestCsv;
+import org.hsqldb.lib.StringUtil;
+import org.joda.time.DateTime;
+
+import javax.annotation.Nullable;
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@IdClass(ExpectedFileKey.class)
+@Table(name = "expected_datasets")
+public class ExpectedDataset {
+    // https://docs.jboss.org/hibernate/orm/5.6/userguide/html_single/Hibernate_User_Guide.html#schema-generation
+
+    // most lengths from easy-dtap/provisioning/roles/easy-fs-rdb/templates/create-easy-db-tables.sql
+    // doi length as in dd-dtap/shared-code/dataverse/scripts/database/create/create_v*.sql
+
+    @Id
+    @Column(length = 255)
+    private String doi;
+
+    @Column(name = "accessible_to")
```

Comment:
there is no such thing as accessible_to for a dataset. we could record the AccessCategory